components/eamxx/src/share/util/scream_vertical_interpolation_impl.hpp

```diff
@@ -46,8 +46,8 @@ void apply_interpolation_impl_1d(
   auto mask   = Kokkos::subview(mask_col,  Kokkos::pair<int,int>(0,num_tgt_packs));
 
   // The input/output data and x_src/x_tgt data should match in the appropriate size, respectively.
-  EKAT_KERNEL_REQUIRE_MSG(x_tgt.size() == output.size(),"Error! vertical_interpolation::perform_checks " + std::to_string(x_tgt.size()) + " != " + std::to_string(output.size()) + ".");
-  EKAT_KERNEL_REQUIRE_MSG(x_src.size() == input.size(), "Error! vertical_interpolation::perform_checks " + std::to_string(x_src.size()) + " != " + std::to_string(input.size()) + ".");
+  EKAT_KERNEL_REQUIRE(x_tgt.size() == output.size());
```

Comment:
This is fine, but notice that you _could_ use `EKAT_KERNEL_REQUIRE_MSG` passing a message that gives the user a bit more context. The catch is that it must be a single "blah" C-style string...